core/types/transaction.go

```diff
@@ -141,13 +145,19 @@ func (tx *Transaction) IntoMutable() (*MutableTransaction, error) {
 	return mutable, nil
 }
 
+// TODO deserialize shard version will failed while use gettxinfo cmd
 func (tx *Transaction) deserializationUnsigned(source *common.ZeroCopySource) error {
 	var irregular, eof bool
 	tx.Version, eof = source.NextByte()
 	var txtype byte
 	txtype, eof = source.NextByte()
 	tx.TxType = TransactionType(txtype)
 	tx.Nonce, eof = source.NextUint32()
+	if tx.Version >= VERSION_SUPPORT_SHARD {
```

Comment:
In transaction-builder, 
set default version of transaction to  VERSION_SUPPORT_SHARD?